index.html

```diff
@@ -30,56 +30,84 @@
       <div class="header-photo"></div>
       <img class="header-logo"src="assets/images/logo.png"/>
       <div class="header-social-media-wrapper">
-        <div class="header-social-media-icon">
+        <div class="header-social-media-icon fb">
           <i class="fab fa-facebook-f"></i>
         </div>
-        <div class="header-social-media-icon">
+        <div class="header-social-media-icon tw">
           <i class="fab fa-twitter"></i>
         </div>
-        <div class="header-social-media-icon">
+        <div class="header-social-media-icon ig">
           <i class="fab fa-instagram"></i>
         </div>
       </div>
-    </div>
   </header>
-  <div class="inner-wrapper">
   <section class="pets-love-wrapper">
-    <span class="pets-love-heading">
-      <h4 class="main-small-heading">Pet's Love</h4>
-     <h1 class="largest-heading">Russian<span>Blue Cat</span></h1>
-    </span>
+    <div class="inner-wrapper">
+     <div class="pets-love-heading">
+      <div class="pets-love-heading-inner-wrapper">
+        <h4 class="main-small-heading">Pet's Love</h4>
+        <h1 class="largest-heading">Russian<span>Blue Cat</span></h1>
+      </div>
+    </div>
     <div class="pets-love-photo">
      <img src="assets/images/the-cat-british.jpeg"/>
     </div>
-    <div class="pets-love-description">
+    <div class="pets-love-description-wrapper">
       <div class="main-small-description">
-      <p>
+      <p class="pets-love-description-first">
         <span>The first step to making your cat famous is</span>
         <span>by building a steady fan base on social</span>
         <span>media! You can start by following other</span>
         <span>animal influencers and learn from what</span>
         <span>they do.</span>
       </p>
-      <p>
+      <p class="pets-love-description-last">
         <span>Don’t stop at clicking photos of your cat in</span>
         <span>fun poses, wearing a cape and looking like</span>
         <span>he’s ready to take over the world! Push the</span>
         <span>creative boundaries to bring..</span>
       </p>
       </div>
-    </section>
+     </div>
     </div>
-    <div class="inner-wrapper-small-margin">
+    </section>
     <section class="dogs-cats-birds-rabbits-wrapper">
-     <div class="dogs-cats-birds-rabbits-photo"></div>
-     <div class="dogs-cats-birds-rabbits-rectangle-wrapper">
-      <div class="dogs-cats-birds-rabbits-rectangle-up"></div>
-      <div class="dogs-cats-birds-rabbits-rectangle-up"></div>
-      <div class="dogs-cats-birds-rabbits-rectangle-down"></div>
-      <div class="dogs-cats-birds-rabbits-rectangle-down"></div>
+      <div class="inner-wrapper dogs-cats-birds-rabbits">
+     <div class="dogs-cats-birds-rabbits-photo">
+       <img src="assets/images/dalmatian.jpeg"/>
      </div>
+     <div class="dogs-cats-birds-rabbits-box-wrapper">
+      <div class="dogs-cats-birds-rabbits-box dogs-cats-box">
```

Comment:
klasa dogs-cats-box jest do wywalenia. Nie robimy takich rzeczy poprzez top: 10%.
Zwróć uwagę na .pets-love-wrapper i zastanów się jak to **ŁADNIE** zrobić.

Reply:
Nie Poprawione